test/e2e/auth/audit.go

```diff
@@ -630,7 +631,18 @@ var _ = SIGDescribe("Advanced Audit", func() {
 			expectedEvents = append(expectedEvents, t.events...)
 		}
 
-		expectAuditLines(f, expectedEvents)
+		// The default flush timeout is 30 seconds, therefore 5 minutes should be enough to wait for
```

Comment:
What I really meant with my previous comment: don't we flush regularly even in batched mode? 5min sounds a lot.

Reply:
Sorry, I clarified the comment: we're waiting for 5 minutes to avoid flakes, in most cases, one retry (30 seconds) should be enough